trackman/templates/library/labels.html

```diff
@@ -19,6 +19,14 @@ <h1>Library</h1>
     <li class="nav-item"><a class="nav-link" href="{{ url_for('trackman_library.fixup') }}">Fixup</a></li>
 </ul>
 
+<!-- TODO beautify -->
+<h2>search<h2>
+    <p>Only searches exact matches</p>
+    <form action=label>
+	<label for="label">Label: </label><input type="text" id="label" name="label"/>
```

Comment:
Please move the space in between the label and input. Also, the input is missing Bootstrap classes; please refer to other forms for an example.